web_widget_html_markdown/static/src/js/web_widget_html_markdown.js

```diff
@@ -0,0 +1,861 @@
+                });
+            } else {
+            var _super = this._super.bind(this);
+            // we must setvalue with force change. 
+            // or it won't pick up changes in the 
+            // switch: USECASE: we modify in MD, then switch to HTML, then save.
+            // to refresh the Wysiwyg in case we are coming from
+            // markdown and no changes have been made in HTML 
+            // (debounce widget considers this case as "Unedited" and does not save).
+            this._setValue(this.value, {forceChange: true});
+            return this.wysiwyg.saveModifiedImages(this.$content).then(function () {
+                return self.wysiwyg.save(self.nodeOptions).then(function (result) {
+                    self._isDirty = true;
+                    _super();
+                });
+            });
+            }
+        },
+        /**
+         * @override
+         */
+        isSet: function () {
+            var value = this.value && this.value.split('&nbsp;').join('').replace(/\s/g, ''); // Removing spaces & html spaces
+            return value && value !== "<p></p>" && value !== "<p><br></p>" && value.match(/\S/);
+        },
+        /**
+         * @override
+         */
+        getFocusableElement: function () {
+            return this.$target || $();
+        },
+        /**
+         * Do not re-render this field if it was the origin of the onchange call.
+         *
+         * @override
+         */
+        reset: function (record, event) {
+            var value = this.value;
+            this._reset(record, event);
+            if (this.nodeOptions.wrapper) {
+                value = this._wrap(value);
+            }
+            value = this._textToHtml(value);
+            if (!event || event.target !== this) {
+                if (this.mode === 'edit') {
+                    this.wysiwyg._setValue(value, {'forceChange': true});
+                } else {
+                    this.$content.html(value);
+                }
+            }
+            return Promise.resolve();
+        },
+
+        // --------------------------------------------------------------------------
+        // Private
+        // --------------------------------------------------------------------------
+
+        /**
+         * @override
+         */
+        _getValue: function () {
+            if (this.markdownEditor){
+                var value = this._getHtmlValue(this.markdownEditor[0].value);
+            } else {
+                var value = this.$target.val();
+            }
+
+            if (this.nodeOptions.wrapper) {
+                return this._unWrap(value);
+            }
+            return value;
+        },
+        /**
+         * Create the wysiwyg instance with the target (this.$target)
+         * then add the editable content (this.$content).
+         *
+         * @private
+         * @returns {$.Promise}
+         */
+        _createWysiwygIntance: function () {
```

Comment:
Instance not Intance

Reply:
this concerned me  a lot. the code COULD NOT had worked.
but fortunately the error is consistent;

static/src/js/web_widget_html_markdown.js:        _createWysiwygIntance: function () {
static/src/js/web_widget_html_markdown.js:                this._createWysiwygIntance();
static/src/js/web_widget_html_markdown.js:                return this._createWysiwygIntance();


and it does not use any super. 
fortunately it;s just consmetic. will fix after done with operating unit requests.